test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.addressbook.common.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.parser.Parser;
+
+public class UtilsTest {
+    
+    private Utils utils;
+    
+    @Before
+    public void setup() {
+        utils = new Utils();
+    }
+    /**
+     * Check the elementsAreUnique method
+     */
+    
+    @Test
+    public void checkElementsAreUniqueWithoutDuplicates() {
+        ArrayList<Integer> arr = new ArrayList<Integer>();
+        arr.addAll(Arrays.asList(1, 2, 3, 4, 5));
+        assertTrue(utils.elementsAreUnique(arr));
+    }
+    
+    @Test
+    public void checkElementsAreUniqueWithDuplicates() {
```

Comment:
the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.